src/index.js

```diff
@@ -25,9 +30,9 @@ export async function run() {
       await execShellCommand('pacman -Sy --noconfirm tmate');
       tmateExecutable = 'CHERE_INVOKING=1 tmate'
     } else {
-      await execShellCommand(optionalSudoPrefix + 'apt-get update');
-      await execShellCommand(optionalSudoPrefix + 'apt-get install -y openssh-client xz-utils');
-
+      if (core.getInput('install_dependencies') === "true") {
+        await installDependenciesLinux(optionalSudoPrefix);
```

Comment:
There is now "duplicate code" which confuses me a little bit between the default parameter in the `installDependenciesLinux` function and the `optionalSudoPrefix` variable.

I think passing a boolean into the function if its sudo or passing the prefix directly into the function would be cleaner.

Reply:
Additionally, why even bother refactoring this? Instead, the `else` could be turned into `else if (core.getInput("install-dependencies") === "true")`. Please note that I implicitly also addressed two more issues for consistency:

- we already use kebap-case in the action, not snake case (i.e. `-` instead of `_` in the input name)
- we seem to use double quotes rather consistently, not single quotes.